### ISSUE TITLE
DB backup utility doesn't handle unicode output from iterdump

### ISSUE THREAD

**Issue by alanchenz** [learningequality/kolibri#2875]
### Observed behavior

When running kolibri, if database backup runs, it returns an error saying that charmap codec cannot encode character '\u259'.

### Expected behavior

Backup should be created without errors

### User-facing consequences

Could prevent a usable backup from being created

### Errors and logs

Command Line:
```
Traceback (most recent call last):
  File "C:\Users\alanc\AppData\Local\Programs\Python\Python35\Lib\runpy.py", lin
e 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\alanc\AppData\Local\Programs\Python\Python35\Lib\runpy.py", lin
e 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\alanc\Kolibri\kolibri\kolibri\__main__.py", line 8, in <module>
    main(args=sys.argv[1:])
  File "C:\Users\alanc\Kolibri\kolibri\kolibri\utils\cli.py", line 632, in main
    initialize(debug=debug)
  File "C:\Users\alanc\Kolibri\kolibri\kolibri\utils\cli.py", line 200, in initi
alize
    backup = dbbackup(version)
  File "C:\Users\alanc\Kolibri\kolibri\kolibri\core\deviceadmin\utils.py", line
90, in dbbackup
    f.write(line)
  File "C:\Users\alanc\AppData\Local\Programs\Python\Python35\Lib\encodings\cp12
52.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u0259' in position
104: character maps to <undefined>
```

### Steps to reproduce

Run yarn install and/or yarn run devserver on Windows 10 through Git bash.

### Context

 * Kolibri version: develop
 * Operating system: Windows 10
 * Browser: NA

**Comment by indirectlylit** [comment]
cc @benjaoming

**Comment by benjaoming** [comment]
Thanks for the report @alanchenz! If you're able to test #2952 that'd be great, we can merge it all the quicker. Am quite sure it will fix it.

**Comment by alanchenz** [comment]
I get a TypeError expecting byte-objects (TypeError: a bytes-like object is required, not 'str'), but it seems to be receiving strings. Is there anyting else I need to change besides changing w to wb in that utils.py file (along with removing db.py)?

**Comment by indirectlylit** [comment]
any possible relationship to this issue, which also references `charmap`?

https://github.com/learningequality/kolibri/issues/2541

**Comment by benjaoming** [comment]
@alanchenz judging from another of your issues, you are running with Python 3.5, right? Not that this should be a problem, just trying to sort out if we are looking at a Python 2-related Unicode error or not.

**Comment by benjaoming** [comment]
It seems this is caused by a console environment setting a different encoding. But we're having some funky test failures when forcing the encoding to UTF-8.

@alanchenz will let you know once it seems ripe for testing! The change to "binary" file mode seems to be breaking stuff, so trying out a new method with forced encoding.

**Comment by alanchenz** [comment]
Yeah I’m using Python 3.5. Also just want to confirm if this is a problem for other people on Windows 10. I wonder if using Git Bash is causing the unicode problems.

**Comment by indirectlylit** [comment]
@alanchenz mind uploading the DB that it's failing on?

**Comment by rtibbles** [comment]
I wonder if this has any relation to #2994

**Comment by radinamatic** [comment]
The issue reported on community forums in #2994 refers to a fresh install, why would a DB backup even be a thing in that case? I thought we set it up to run only at version upgrades...?

**Comment by rtibbles** [comment]
@radinamatic I am wondering if they might have the same root cause, Unicode/Python-wise - but not triggered by the same code.

**Comment by radinamatic** [comment]
Gotcha... 

That user is running Windows in German, in case it makes any difference. I tried with Win VM in Spanish, but couldn't replicate, might try in German too, just in case...

**Comment by radinamatic** [comment]
#2994 Replicated in [Win 10 with UI in German](https://github.com/learningequality/kolibri/issues/2994#issuecomment-353688421).

**Comment by benjaoming** [comment]
@radinamatic beware that these are two different errors both caused by the `cp1252` encoding on Windows.

The one with dbbackup/dbrestore can only be provoked either by changing versions so `dbbackup` is invoked automatically, or you can test it manually by running `kolibri manage dbbackup`

**Comment by benjaoming** [comment]
Fixed in #2955
